wazo_call_logd/cel_interpretor.py

```diff
@@ -378,20 +382,32 @@ def interpret_xivo_incall(self, cel, call):
         call.set_tenant_uuid(extra['extra'])
         return call
 
-    def interpret_xivo_outcall(self, cel, call):
-        call.direction = 'outbound'
-
+    def interpret_xivo_outcall(self, cel, call: RawCallLog):
+        if not call.was_forwarded:
+            call.direction = 'outbound'
+        else:
+            call.destination_details.clear()
+            call.destination_exten = call.extension_filter.filter(cel.cid_num)
+            call.destination_name = cel.cid_name
+            logger.debug('call was forwarded, identified a new external destination')
         return call
 
-    def interpret_xivo_user_fwd(self, cel, call):
+    def interpret_xivo_user_fwd(self, cel, call: RawCallLog):
+        call.was_forwarded = True
+        extra = re.match(EXTRA_USER_FWD_REGEX, cel.extra)
+
+        # Replace destination_exten here because WAZO_USER_MISSED_CALL event
+        # doesn't produce the correct destination extension
```

Comment:
Can't that be fixed directly in `WAZO_USER_MISSED_CALL`?

Reply:
It is, but requires (possibly) more work and might introduce unforeseen side-effects because we would be modifying the dialplan directly.  Any other system that would use `WAZO_USER_MISSED_CALL` would now received possibly incorrect data.  I've opened another ticket to take a look

Reply:
https://wazo-dev.atlassian.net/browse/WAZO-3933